src/views/identity-checks-completed/identity-checks-completed.njk

```diff
@@ -89,7 +89,7 @@
        {{ govukButton({
          text: i18n.Continue,
          attributes: {
-           "data-event-id": "CONTINUE - IDENTITY CHECKS"
+           onClick="_paq.push(['trackGoal', '16'])"
```

Comment:
syntax is incorrect, it should be
"onClick": "_paq.push(['trackGoal', '16'])"

Reply:
Yes made that correct. Thanks